libcontainer/console_linux.go

```diff
@@ -5,18 +5,23 @@ import (
 	"os"
 	"syscall"
 	"unsafe"
+
+	"github.com/docker/docker/pkg/term"
```

Comment:
I think there is a requirement to not depend on `docker/docker` to avoid circular dependencies. May be betetr to copy the one function into this repo.